File: ruby/samples/basic.rb
# Load the gem
require 'intrinio-sdk'

# Setup authorization
Intrinio.configure do |config|
  config.api_key['api-key'] = 'YOUR API KEY'
end

company_api = Intrinio::CompanyApi.new

begin
  results = company_api.filter_companies()
  results.each do |company|
    puts company.name
  end
rescue Intrinio::ApiError => e
  puts "Exception when calling CompanyApi->filter_companies: #{e}"
end

File: javascript/samples/basic.js
var intrinio = require('intrinio');
intrinio.ApiClient.instance.authentications['HttpHeaderApiKey'].apiKey = "YOUR API KEY";

var company_api = new intrinio.CompanyApi();
var opts = {};

company_api.getAllCompanies(opts).then(function(data) {
  console.log('API called successfully.');
  data.forEach(function(company) {
    console.log(company.name);
  });
}, function(error) {
  console.error(error);
});
